example/index.html

```diff
@@ -189,6 +189,11 @@
         accessors: {
           icon: 'nodeType'
         },
+        label: function (selection) {
```

Comment:
I don't want to pass a boolean to control this. This gives you full control if you want to change the label contents.